src/Networks/Rutanio/Rutanio/Rutanio.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
I think you dont need that its set on the global props (but up to you)

Reply:
We have removed them from Rutanio project.